.gitattributes

```diff
@@ -1,7 +1,9 @@
 * text=input
 
-*.md eol=lf
-*.json eol=lf
+*.bat text eol=crlf
+*.md text eol=lf
+*.json text eol=lf
+*.player text eol=lf
```

Comment:
Some of these player files are from gridlabd, do we know if gridlab can handle either line ending?  I presume it could but might be something to at least consider

Reply:
Given that GridLab-D support for Windows is using MinGW, it's more likely to have trouble with Windows style (crlf) line endings or [old-macintosh (cr) line endings](https://github.com/gridlab-d/gridlab-d/issues/846).

It [looks like they expect \n but not \r for blank lines](https://github.com/gridlab-d/gridlab-d/blob/f56223aa6e10dd130668ec9f8092da96f5d3f895/tape/player.c#L318-L321).

Reply:
All of the [recorder](https://github.com/gridlab-d/gridlab-d/blob/f56223aa6e10dd130668ec9f8092da96f5d3f895/tape/recorder.c#L147-L170) uses `\n` line endings without `\r`.

If their player gets broken by files only having a `\n`, I think that's a pretty major bug.